MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -1527,8 +1527,7 @@ public static ReinforcementEligibilityType getReinforcementType(int forceID, Str
         }
 
         // otherwise, the force requires support points / vps to deploy
```

Comment:
Change in code make the commentary above it wrong, should be changed to match the code